Source/ConvolutionFunctions/arm_convolve_get_buffer_sizes_s4.c

```diff
@@ -81,6 +136,7 @@ int32_t arm_convolve_wrapper_s4_get_buffer_size(const cmsis_nn_conv_params *conv
     {
```

Comment:
An else if() guarded by ARM_MATH_MVEI would save code duplication and make the maintenance easier here.

Reply:
This would reduce duplication but would also either break api or break the idea of having a wrapper function - decided to leave it for now